client/queries/queries.ts

```diff
@@ -1,9 +1,130 @@
-import { MemcmpFilter, PublicKey } from "@solana/web3.js";
+import { Connection, MemcmpFilter, PublicKey } from "@solana/web3.js";
 import bs58 from "bs58";
 import { Event } from "../accounts";
+import { PROGRAM_ID } from "../programId";
 
 export const GET_MULTIPLE_ACCOUNTS_LIMIT = 99;
 
+interface QueryableAccountClass {
+  discriminator: Buffer;
+  fetchMultiple(
+    c: Connection,
+    addresses: PublicKey[],
+  ): Promise<Array<Event | null>>;
+  fetch(
+    c: Connection,
+    address: PublicKey,
+  ): Promise<Event | null>;
+}
```

Comment:
Not super keen on this but it was the only way I could make it work. If the auto generated code created an account interface or something that all the accounts respected then maybe it would be easier. At least here typescript _should_ complain if you try to pass in a class that doesn't fulfil this interface.